doc/en/installation/index.rst

```diff
@@ -30,3 +30,7 @@ Finally, you can install the platform.
     from_packages
     from_VM
     from_sources
+
+To quickly test Centreon, you can from a CentOS or Red Hat 7.x server run the following command: ::
+
+    # curl -L https://raw.githubusercontent.com/centreon/centreon/master/unattended.sh | sh
```

Comment:
shorten url with https://git.io? (or another tool)
```suggestion
    # curl -L https://git.io/Jeclg | sh
```

Reply:
IMHO we should not use a URL shortener because 1) it will not be canonical and 2) it prevents users from knowing the source URL of the shell script that will be executed on the machine.